styleguide/derek/pattern-components/additional-resources/_partials/additional-resource.ejs

```diff
@@ -13,15 +14,20 @@ PDF: fa-file-pdf-o
 API/Code: fa-file-code-o
 Support/KC: fa-wrench
 -->
-<div class="additional-resource">
+<div class="resourceWrapper resource-<%- type -%>Icon">
   <a href="javascript:void(0)">
```

Comment:
Let's apply a `track-click` class to the anchor to remind the us that tracking on this pattern is considered.